src/fbjava-tests/src/main/java/example/Procedures.java

```diff
@@ -174,4 +178,11 @@ public boolean fetch() throws Exception
 			}
 		};
 	}
+
+	public static ResultSet testResultSet(long[] o1, String[] o2) throws SQLException {
+		Connection con = DriverManager.getConnection("jdbc:new:connection:");
```

Comment:
How is this `Connection` closed?

Reply:
It's a mistake. `jdbc:default:connection:` should be used here.

Reply:
The design of the feature looks bugged for me. How statements will be closed when a `ResultSet` is returned?

Reply:
`jdbc:default:` calls `freeStatements` method when closed. That's not enough?

Reply:
No. Allowing returning ResultSet, one could return ResultSet from others connections which will make easy to have leaks.

I think the code should be adapted to the user explicitly use the wrapper class with some way to it close statement/connection when closed.